package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@walletconnect/chat-client",
   "description": "WalletConnect Chat Client",
-  "version": "0.1.9",
+  "version": "0.2.0",
```

Comment:
Since the chat SDK is already in alpha/being experimented with by some people, could you add a git tag for this (`git tag 0.2.0` to apply to last commit) so we can generate release notes for this change please? 

Should then just be a matter of `Releases > Generate release notes`. Will add a tag for `0.1.9` now if there's not one yet.

Reply:
Tagged `0.1.9` and pushed the tag up ✅